spec/features/posts_index_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:user) do
+    User.create(name: 'Adnan', photo: 'https://unsplash.com/photos/ABC123', posts_counter: 5)
+  end
+
+  let!(:post) do
+    Post.create(title: 'My First Post', text: 'This is my first post', author: user, comments_counter: 0,
+                likes_counter: 0)
+  end
+
+  let!(:comment) do
```

Comment:
- [ ] Please write featured tests for the user post show page. Kindly see the attached image below for the tests you are expected to write 👇 
![image](https://github.com/MarcoDDM/blog-app/assets/46839548/1a86d021-29f5-4bd2-838e-aae84cc43703)